### ISSUE TITLE
i18next-parser don't handle a key with multi-line

### ISSUE THREAD

**Issue by jare-talend** [i18next/i18next-parser#89]
- Is this a bug or a feature request? bug
- What version are you running (`0.x` or `1.x`)? 1.x
- Which runner (cli, gulp...)? cli

## What I'm trying to do?
I want to parse a file with options on multiline

e.g.

src/test.js

```
t('KEY1', {
        defaultValue: '{{var}} translated',
});
t('KEY2', {  defaultValue: '{{var}} translated'});

```

## How am I doing it?

*Replace this section with your config file or cli command*

Here is my config file:

```
{
  output: 'locales'
}
```

The command I'm running:

```
i18next src/**/*.js -c ./i18n.json
```

## What was the result?
```
{
  "KEY2": " translated"
}
```
*Replace this with a description of the result you are getting*

## What was I expecting?

```
{
  "KEY1": "{{var}} translated"
  "KEY2": " translated"
}
```
the 0.x version works well

**Comment by karellm** [comment]
This will require a rewrite using a parser. This is done for Javascript but now I need to rewrite it for React to have all the test to pass.

**Comment by karellm** [comment]
Progress can be followed here: https://github.com/i18next/i18next-parser/pull/92

**Comment by karellm** [comment]
This should be addressed in `1.0.0-beta13`
